front/cypress/e2e/survey_builder/create_and_fill_survey.cy.ts

```diff
@@ -294,41 +267,14 @@ describe.skip('Survey builder', () => {
     );
   });
 
-  it('does not allow editing survey fields in builder when responses have started coming in', () => {
```

Comment:
Not 100% sure this was the reason I removed it, but looking at it now I think the test might not be doing what it describes (nothing happens after submitting the response, while the description implies a check should happen to see if the survey can be edited). Also the test "shows survey results for multiple choice and linear scale fields" seems to currently cover this case.